Lib/test/test_uuid.py

```diff
@@ -549,7 +558,10 @@ def check_node(self, node, requires=None):
 
     @unittest.skipUnless(os.name == 'posix', 'requires Posix')
     def test_ifconfig_getnode(self):
-        node = self.uuid._ifconfig_getnode()
+        if _notAIX:
+            node = self.uuid._ifconfig_getnode()
+        else:
+            self.skipTest('because AIX "ifconfig" does not provide macaddr')
```

Comment:
You should use @unittest.skipIf(AIX, "...").

Reply:
much nicer. done